www/js/services/user_service.js

```diff
@@ -33,7 +33,11 @@ app.service('UserService', function($q, $http, $httpParamSerializer, RequestPara
         throw "Seller does not have deviceID";
       }
       this.seller = seller;
-      $rootScope.$emit('sellerLogin');
+
+      $timeout(function() {
+        $rootScope.$emit('sellerLogin');
+      },1000);
```

Comment:
no. This for loading the seller from localStorage. It's firing an event called 'sellerLogin', to let all interested that the seller is logged in now.

I didn't do anything on the preference "stay signed in", but if it's for keep the seller logged in or not,  we can use this.